lib/withInNativeApp.js

```diff
@@ -96,10 +96,12 @@ export const postMessage = !inNativeAppBrowser
         typeof msg === 'string' ? msg : JSON.stringify(msg),
         '*'
       )
-  : msg =>
+  : window.ReactNativeWebView
+  ? msg =>
       window.ReactNativeWebView.postMessage(
         typeof msg === 'string' ? msg : JSON.stringify(msg)
       )
+  : msg => console.warn('postMessage unavailable', msg)
```

Comment:
what's this about?

Reply:
Basically it allows to simulate in app behaviour in chromes device emulator without causing an exception because `ReactNativeWebView` is not available.

I use this to test app layout without opening simulators/emulators:
<img width="1546" alt="Screenshot 2021-02-11 at 10 53 40" src="https://user-images.githubusercontent.com/410211/107622000-707b1080-6c57-11eb-8b8c-11ec62cf4de7.png">

Reply:
nice, way quicker than starting up a simulator!

Reply:
Is this documented somewhere?

Reply:
Added a note to the readme: https://github.com/orbiting/republik-frontend/pull/510/commits/9a9ad4ab91718170f4930d4c896e33d1a3a6344d